### ISSUE TITLE
Option correction is too eager

### ISSUE THREAD

**Issue by cheshire** [wfxr/code-minimap#6]
```
code-minimap -- compiler/xla/service/copy_insertion.cc
error: The subcommand 'compiler/xla/service/copy_insertion.cc' wasn't recognized
        Did you mean 'completion'?

If you believe you received this message in error, try re-running with 'code-minimap -- compiler/xla/service/copy_insertion.cc'
```

**Comment by wfxr** [comment]
Hi @cheshire Thanks for your feedback!

This bug was caused by `clap` crate which help `code-minimap` parse cli arguments.

Related issue: https://github.com/clap-rs/clap/issues/878

These is a workaround you can use before this is fixed:
```
code-minimap ./compiler/xla/service/copy_insertion.cc
```

**Comment by tobiasvl** [comment]
This seems to have been fixed upstream?

**Comment by wfxr** [comment]
@tobiasvl Yes but still not been released.

**Comment by wfxr** [comment]
:tada: `clap` just released v3.0.0: https://github.com/clap-rs/clap/commit/c01ebbac17e3fdaf3b1d0bfb84051cd701babb7c, so this issue has been fixed by #39.
